src/sentry/api/endpoints/organization_member/index.py

```diff
@@ -371,27 +375,39 @@ def post(self, request: Request, organization) -> Response:
                 status=400,
             )
 
-        with transaction.atomic(router.db_for_write(OrganizationMember)):
-            # remove any invitation requests for this email before inviting
-            existing_invite = OrganizationMember.objects.filter(
-                Q(invite_status=InviteStatus.REQUESTED_TO_BE_INVITED.value)
-                | Q(invite_status=InviteStatus.REQUESTED_TO_JOIN.value),
-                email=result["email"],
-                organization=organization,
-            )
-            for om in existing_invite:
-                om.delete()
-
-            om = OrganizationMember(
-                organization=organization,
-                email=result["email"],
-                role=result["role"],
-                inviter_id=request.user.id,
-            )
+        if superuser_has_permission(request):
```

Comment:
if this is in the admin panel, can we allow staff to hit this endpoint? people with access to admin may not have superuser in the future cc @schew2381

Reply:
for instance, we could assert that somebody is active staff if they're trying to pass in a `user_id`, and use the existence of the `user_id` to determine which flow to hit

Reply:
I see, just to make sure I understand the issue here correctly, in the future a user with _admin might not have superuser correct?

Reply:
a user with _admin = a user with `is_staff` so we can check `is_active_staff`. i also see this endpoint already has `MemberAndStaffPermission`